src/components/chips/js/chipsController.js

```diff
@@ -193,6 +260,41 @@
    */
   MdChipsCtrl.prototype.configureMdAutocomplete = function(mdAutocompleteCtrl) {
     this.mdAutocompleteCtrl = mdAutocompleteCtrl;
-    // TODO(typotter): create and register a selectedItem watcher with mdAutocompleteCtrl.
+    this.mdAutocompleteCtrl.registerSelectedItemWatcher(angular.bind(this, this.mdAutocompleteSelectedItemWatcher));
+  };
+
+
+  MdChipsCtrl.prototype.mdAutocompleteSelectedItemWatcher = function(newItem, oldItem) {
+    if (!!newItem && newItem !== oldItem) {
```

Comment:
You never need !! in conditionals like if, while, etc. You only would use !! if you have a function that returns or is expecting a boolean and nothing else.